src/o365/spo/commands/file/file-get.ts

```diff
@@ -0,0 +1,277 @@
+import auth from '../../SpoAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import * as request from 'request-promise-native';
+import {
+  CommandOption,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import { ContextInfo } from '../../spo';
+import Utils from '../../../../Utils';
+import Auth from '../../../../Auth';
+import fs = require('fs');
+import * as path from 'path';
+import { FileProperties } from './FileProperties';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  url: string;
+  id: string;
+  asString: boolean;
+  asListItem: boolean;
+  asFile: boolean;
+  fileName: string;
+  path: string;
+}
+
+class FileGetCommand extends SpoCommand {
```

Comment:
Since this is an SPO-command, could you prefix the class name with `Spo` for consistency with other commands?

Reply:
Included Spo in class name